roles/sequencing-report-ws/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- block:
+  - include: install.yml
+  - include: install_config.yml
+
+  - name: add static folders to be created
+    set_fact:
+      static_folders: "{{ static_folders + [arteria_service_log_dir, runfolder_path, project_path, staging_path, delivery_links] }}" # TODO check if we need this
```

Comment:
don't think you need all of these, right? should only need the dirs the service is actually reading/writing to/from

Reply:
Yeah I wasn't quite sure what these locations were for, I'm guessing we only need `arteria_service_log_dir` and `runfolder_path` for the sequencing report service? Can you confirm this @matrulda?